tests/node/rpc/enums/VehicleDataTypeTests.js

```diff
@@ -85,7 +87,8 @@ describe('VehicleDataTypeTests', function () {
         Validator.assertNotNullUndefined(enumVehicledataFuellevel, 'VEHICLEDATA_FUELLEVEL returned null.');
         Validator.assertNotNullUndefined(enumVehicledataFuellevelState, 'VEHICLEDATA_FUELLEVEL_STATE returned null.');
         Validator.assertNotNullUndefined(enumVehicledataFuelconsumption, 'VEHICLEDATA_FUELCONSUMPTION returned null.');
-        Validator.assertNotNullUndefined(enumVehicledataExterntemp, 'VEHICLEDATA_EXTERNTEMP returned null.');
+        Validator.assertNotNullUndefined(enumVehicledataClimatedata, 'VEHICLEDATA_FUELCONSUMPTION returned null.');
```

Comment:
line 90 should be `Validator.assertNotNullUndefined(enumVehicledataClimatedata, 'VEHICLEDATA_CLIMATEDATA returned null.');`

Reply:
Fixed in d1e6617